proposals/dao/tip_121b.ts

```diff
@@ -36,10 +41,13 @@ Steps:
 const fipNumber = 'tip_121b';
 
 const dripPeriod = 3600; // 1 hour
-const dripAmount = ethers.utils.parseEther('2500000'); // 2.5m Fei
+const dripAmount = ethers.utils.parseEther('1000000'); // 1m Fei
 
-const rariMerkleRedeemerInitialBalance = ethers.utils.parseEther('5000000'); // 5m Fei
-const merkleRedeemerDripperInitialBalance = ethers.utils.parseEther('45000000'); // 45m Fei
+const total = parseEther('12545744'); // 12.545M Fei total
```

Comment:
Dripper won't be able to drip all this no? It needs to be an integer multiple of the dripAmount?

Reply:
Only the dripper needs to be a multiple